web/src/components/molecules/MyIntegration/WebhookForm/index.tsx

```diff
@@ -0,0 +1,105 @@
+import styled from "@emotion/styled";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Checkbox from "@reearth-cms/components/atoms/Checkbox";
+import Col from "@reearth-cms/components/atoms/Col";
+import Divider from "@reearth-cms/components/atoms/Divider";
+import Form from "@reearth-cms/components/atoms/Form";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import Input from "@reearth-cms/components/atoms/Input";
+import Row from "@reearth-cms/components/atoms/Row";
+import { useT } from "@reearth-cms/i18n";
+
+// TODO: will be changed when when hooks are added
+const options = [
+  { label: "Create", value: "Create" },
+  { label: "Update", value: "Update" },
+  { label: "Delete", value: "Delete" },
+  { label: "Accessed by API", value: "Accessed by API" },
+];
```

Comment:
```suggestion
const options = [
  { label: t("Create"), value: "Create" },
  { label: t("Update"), value: "Update" },
  { label: t("Delete"), value: "Delete" },
  { label: t("Accessed by API"), value: "Accessed by API" },
];
```
Labels should be wrapped in `t()` right? or are these labels not final yet?
If it is, please move the options array into the component and wrap the labels in `t()`

Reply:
https://github.com/reearth/reearth-cms/blob/e95d43e9690f111c5e16219c37265df9a35c5b7a/web/src/components/molecules/MyIntegration/WebhookForm/index.tsx#L17-L22